cinema/serializers.py

```diff
@@ -68,13 +78,53 @@ class Meta:
             "movie_title",
             "cinema_hall_name",
             "cinema_hall_capacity",
+            "tickets_available"
         )
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ("movie_session", "row", "seat")
```

Comment:
this will have redundant movie session ids in each item of taken seats when we have this already one level above (see the task for example)